Dockerfile.template

```diff
@@ -6,14 +6,18 @@ FROM cimg/%%PARENT%%:2022.01
 
 LABEL maintainer="Community & Partner Engineering Team <community-partner@circleci.com>"
 
-ENV GO_VERSION=%%MAIN_VERSION%%
+ENV GO_VER="%%VERSION_FULL%%"
```

Comment:
VER and VERSION_FULL are the pattern/variables we should be using. The others are deprecated.